resources/views/student-council/economic-committee/app.blade.php

```diff
@@ -126,6 +122,14 @@
                             Azok közül, akik fizettek, minden bentlakó után a műhely {{config('custom.kkt')}} * {{config('custom.workshop_balance_resident')}}, minden bejáró után {{config('custom.kkt')}} * {{config('custom.workshop_balance_extern')}} forintot kap.
                             (Ha egy collegistának több műhelye is van, a műhelyei megosztoznak az összegen.)
                         </blockquote>
+                        @can('calculateWorkshopBalance', \App\Models\Checkout::studentsCouncil())
```

Comment:
\App\Models\Checkout::class is enough in this case.

Reply:
Thanks, fixed it